testutils/src/main/scala/za/co/absa/ComparisonJob.scala

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2018 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa
+
+import org.apache.spark.sql._
+import org.apache.spark.SparkContext
+import org.apache.spark.sql.types.{StructField, StructType}
+import za.co.absa.enceladus.standardization.CmdConfig
+
+object ComparisonJob {
+  def getReader(cmd: CmdConfig)(implicit sparkSession: SparkSession): DataFrameReader = {
+    val dfReader = sparkSession.read.format(cmd.rawFormat)
+    // applying format specific options
+    val  dfReader4 = {
+      val dfReader1 = if (cmd.rowTag.isDefined) dfReader.option("rowTag", cmd.rowTag.get) else dfReader
+      val dfReader2 = if (cmd.csvDelimiter.isDefined) dfReader1.option("delimiter", cmd.csvDelimiter.get) else dfReader1
+      val dfReader3 = if (cmd.csvHeader.isDefined) dfReader2.option("header", cmd.csvHeader.get) else dfReader2
+      dfReader3
+    }
+    if (cmd.rawFormat.equalsIgnoreCase("fixed-width")) {
+      val dfReader5 = if (cmd.fixedWidthTrimValues.get) dfReader4.option("trimValues", "true") else dfReader4
+      dfReader5
+    } else dfReader4
+  }
+
+  def removeMetadataFromSchema(schema: StructType): StructType = {
+    StructType(schema.map{ f =>
+      StructField(f.name, f.dataType, f.nullable)
+    })
+  }
+
+  def main(args: Array[String]): Unit = {
+    val cmd = CmdConfig.getCmdLineArguments(args)
```

Comment:
I'd suggest a fail-fast check in case refPath and stdPath pointing to he same location.